tools/unit_tests.py

```diff
@@ -43,12 +43,12 @@ def run_unit_test(deno_exe, permStr, flags=None):
 # tests by the special string. permW0N0 means allow-write but not allow-net.
 # See js/test_util.ts for more details.
 def unit_tests(deno_exe):
-    run_unit_test(deno_exe, "permW0N0E0R0")
-    run_unit_test(deno_exe, "permW1N0E0R0", ["--allow-write"])
-    run_unit_test(deno_exe, "permW0N1E0R0", ["--allow-net"])
-    run_unit_test(deno_exe, "permW0N0E1R0", ["--allow-env"])
-    run_unit_test(deno_exe, "permW0N0E0R1", ["--allow-run"])
-    run_unit_test(deno_exe, "permW1N0E0R1", ["--allow-run", "--allow-write"])
+    run_unit_test(deno_exe, "permR1W0N0E0U0", ["--allow-read"])
```

Comment:
I think we should modify the tests so that we explicitly ask for `testPerm({ read: true }, ...)`
so also add `permR0W0N0E0U0` here.